challenges/classes.js

```diff
@@ -38,30 +38,34 @@ const testCuboid = new CuboidMaker({
 
 class CubeMaker extends CuboidMaker {
   constructor(props) {
-  super(props);
+    super(props);
   }
   volume() {
-    if (this.length != this.width || this.length != this.height || this.width != this.height){
-    return 'This is not a cube, please try the CuboidMaker next time';
-    }else if ( this.length === 0 || this.width === 0 || this. height === 0 ) {
+    if (this.length === 0 || this.width === 0 || this.height === 0) {
       return 'Sorry, all sides must be non-zero values.'
-    } else return (this.length **=3);
+    } else if (this.length != this.width || this.length != this.height || this.width != this.height) {
+      return 'This is not a cube, please try the CuboidMaker next time';
+    } else {
+      return (this.length ** 3);
+    }
   }
   surfaceArea() {
-     if (this.length != this.width || this.length != this.height || this.width != this.height) {
-       return 'This is not a cube, please try the CuboidMaker next time';
-     } else if (this.length === 0 || this.width === 0 || this.height === 0) {
-       return 'Sorry, all sides must be non-zero values.'
-     } else return (6 * (this.length **= 2));
+    if (this.length === 0 || this.width === 0 || this.height === 0) {
+      return 'Sorry, all sides must be non-zero values.'
+    } else if (this.length != this.width || this.length != this.height || this.width != this.height) {
+      return 'This is not a cube, please try the CuboidMaker next time';
+    } else {
+      return (6 * (this.length ** 2));
+    }
   }
 
```

Comment:
Starting to get into a little error handling here I see. Awesome! You shall need those skills later down the road.